docs/styleguide.md

```diff
@@ -112,11 +112,11 @@ Using the `Session` and `Cookies` classes as an example:
 ```markdown
 # session
 
-## Methods
+## Static Methods
```

Comment:
`session.fromPartition` is a method of the `session` module, instead of a static method of `Session` class. I'm not sure if it should be marked as `Static` here.

Reply:
yeah, i think it may be better to use a different module example to illustrate this point